stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -0,0 +1,181 @@
+#ifndef STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_LPMF_HPP
+#define STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_LPMF_HPP
+
+#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
+#include <stan/math/prim/scal/meta/partials_return_type.hpp>
+#include <stan/math/prim/scal/meta/operands_and_partials.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/fun/constants.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+#include <stan/math/prim/scal/fun/value_of.hpp>
+#include <stan/math/prim/scal/meta/include_summand.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
+#include <boost/random/bernoulli_distribution.hpp>
+#include <boost/random/variate_generator.hpp>
+#include <cmath>
+#include <string>
+
+using Eigen::Dynamic;
+using Eigen::Matrix;
+
+namespace stan
+{
+namespace math
+{
+
+/**
+     * Returns the log PMF of the Generalized Linear Model (GLM)
+     * with Bernoulli distribution and logit link function.
+     * If containers are supplied, returns the log sum of the probabilities.
+     * @tparam T_n type of integer parameter
+     * @tparam T_x type of the data matrix (design matrix);
+     * this should be an Eigen::Matrix type
+     * @tparam T_beta type of the weight vector
+     * @tparam T_alpha type of the bias vector
+     * @param n integer parameter
+     * @param x design matrix
+     * @param beta weight vector
+     * @param alpha bias vector (in logits)
+     * @return log probability or log sum of probabilities
+     * @throw std::domain_error if theta is infinite.
+     * @throw std::invalid_argument if container sizes mismatch.
+     */
+
+template <bool propto,
+          typename T_n,
```

Comment:
No space between doc and function.

The comment should be lined up above the code.

Pack as many template parameters (and function arguments) on a line as you can up to 80 characters.

Reply:
OK.